File: src/main/java/location/databinding/PlainRecordObject.java
package location.databinding;

/**
 * POJO for parsing the location records from the Google-generated JSON
 * Aforementioned JSON file contains mainly an array of these types of records
 * Properties not used by this program are thrown away at parsing time
 */
public class PlainRecordObject {
    public long timestampMs;
    public int latitudeE7;
    public int longitudeE7;
    public int accuracy;
}

File: src/main/java/location/databinding/PlainRecordObject.kt
package location.databinding

/**
 * Model of the array item in the Google Location takeout JSON file
 * Properties not present here are thrown away at parsing time
 */
data class PlainRecordObject(
        var timestampMs: Long = 0,
        var latitudeE7: Int = 0,
        var longitudeE7: Int = 0,
        var accuracy: Int = 0
)
